README.md

```diff
@@ -102,6 +106,14 @@ Here is an example of the data sent in the message:
 }
 ```
 
+### Defining Telegram handlers
+
+A Telegram handler sends message with the photo that triggered the event.
```

Comment:
Document how to actually obtain the chatIds here. Also it sends the photo and the trigger name as the caption. Make that clear. Include a photo too?

Reply:
Fixed